file-formats/saia/README.md

```diff
@@ -0,0 +1,5 @@
+# SAIA File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :--- | :--- | :--- | :---
+`<name>.saia.json` | 1 | [`zif_aff_saia_v1.intf.abap`](./type/zif_aff_saia_v1.intf.abap) | [`saia-v1.json`](./saia-v1.json) | [`z_aff_example_saia.saia.json`](./examples/z_aff_example_saia.saia.json)
```

Comment:
The abaplint check is working again, the link check is still failing, because an example is linked here, but the file `z_aff_example_saia.saia.json` is not included in this pull request. Maybe a `git add` is missing for the file?